### PR TITLE
fix requests with no token

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

**Comment by luytena** [comment]
@velrest rebase please :)

File: index.js [hunk 1]
```diff
@@ -13,4 +13,14 @@ module.exports = EngineAddon.extend({
...
   },
+  included(...args) {
+    const app = this._findHost(this);
+    app.options["ember-validated-form"] = {
+      theme: "uikit",
+      defaults: {
+        "types/date": "ember-ebau-gwr/components/model-form/date",
+      },
+    };
+    this._super.included.apply(this, args);
```
Comment (luytena) [inline index.js:24]: Does `super.included....` work as well?

Comment (velrest) [reply index.js:24]: No sadly not since we're in a classic `EmberObject` environment.

**Review by luytena — COMMENTED** [review]
Approved

**Pull request merged**